apps/omg_watcher/test/omg_watcher/api/account_test.exs

```diff
@@ -63,5 +66,44 @@ defmodule OMG.Watcher.API.AccountTest do
 
       assert %{blknum: ^blknum, txindex: ^txindex, oindex: ^oindex} = utxo
     end
+
+    @tag fixtures: [:db_initialized]
```

Comment:
I will get rid of this fixture before making it ready to review, but review comments are welcomed.

Reply:
I don't think we're heading to ditch all fixtures, once they already there. Such fixtures are very useful and also keeps the codebase clean. That's my opinion.

Reply:
You are right. I like them to. The problem we have is that if test uses a fixture defined in other app it can be run only in bulk, not as a single test. There might be a problem in how we designed tests. In this case removing fixture was easy, so I did it.